dom.js

```diff
@@ -0,0 +1,33 @@
+var APP = APP || {};
+(function(app) {
+
+    function element(name) {
+        return function() {
+            var e = document.createElement(name);
+            var args = Array.prototype.slice.call(arguments);
```

Comment:
I can change this, but it means having to do stuff like `app.dom.h1(null, null, 'My text')` which gets really noisy in the application logic.